docs/README.md

```diff
@@ -1,10 +1,83 @@
-# Documentation
+# Documentation structure
 
```

Comment:
Maybe a small disclaimer here about thing being WIP and a lot is gonna be missing? :grin:

Reply:
Fixed @Rugvip